### PR TITLE
Xml

### PR DESCRIPTION
**Pull request by mennoo1996**
Created load and save functionality
Added load and save buttons to UI

### CODE REVIEW

File: eenFile.txt [hunk 1]
```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```
Comment (bart-de-jonge) [inline eenFile.txt]: Is this file still necessary, if yes please give a legit name please

File: src/main/java/control/TimelineController.java [hunk 2]
```diff
@@ -93,4 +102,74 @@ private void initializeCameraTimelines() {
             scriptingProject.addCameraTimeline(timelineN);
         }
     }
+    
+    /**
+     * Save the current project state to file.
+     * A file chooser window will be opened to select a file
+     */
+    public void save() {
```
Comment (bart-de-jonge) [inline src/main/java/control/TimelineController.java:110]: I made a save and load (or read and write but you can rename them) function in the ScriptingProject class. Could you move this code to that location since I believe saving and loading belongs to a project and not a timelinecontroller

File: src/main/java/data/Camera.java [hunk 3]
```diff
@@ -1,14 +1,19 @@
...
+import lombok.Setter;
+import lombok.ToString;
 
 /**
  * Created by Bart.
  * Class to store information about cameras.
  */
+@XmlRootElement(name = "camera")
 @EqualsAndHashCode
+@ToString
```
Comment (bart-de-jonge) [inline src/main/java/data/Camera.java:16]: Can some of these annotations be placed on one line (like @EqualsAndHashCode and @ToString)

**Comment by bart-de-jonge** [comment]
Looks good! Please look at the comments, then we can merge

File: src/main/java/data/Camera.java (hunk 3 above)
Comment (mennoo1996) [inline src/main/java/data/Camera.java:16]: No, according to our checkstyle ruleset these annotations should be on seperate lines

File: eenFile.txt (hunk 1 above)
Comment (mennoo1996) [inline eenFile.txt]: No, I'll remove it

File: src/main/java/control/TimelineController.java (hunk 2 above)
Comment (mennoo1996) [inline src/main/java/control/TimelineController.java:110]: Yes, I think you're right. I'll fix it in the next commit(s)

**Comment by bart-de-jonge** [comment]
Nice work
